### PR TITLE
feat: scrollbar for SfSidebar content section

### LINKED ISSUE #1208: [FEATURE] The sidebar needs a scroll bar to scroll through added products.
**Describe how this feature will help us improve**
It is better to display the scrollbar in the sidebar, because users without a scroll wheel mouse cannot view all added products.

**Is your proposal related to a problem?**
Describe the issue or type its number

**Describe the solution you'd considered**
Add scrollbar in the sidebars.

If there are multiple solutions, please present each one separately. Save comparisons for the very end.

**Additional context**
Anything you would like to add but was not sure which section it fits.

### PR DESCRIPTION
**Pull request by AdamPawlinski**
# Related issue
Closes #1208

# Scope of work
scrollbar added to the sidebar

# Screenshots of visual changes
image

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed

### CODE REVIEW

**Comment by pspaczek** [comment]
@AdamPawlinski styles for ::-webkit-scrollbar doesn't work well, maybe we can use SfScrollable like in SfMegaMenu for Search results -> https://storybook.storefrontui.io/?path=/story/organisms-megamenu--search-results. What do you think?

**Comment by AdamPawlinski** [comment]
Yes, you are right, I'll fix that. Thanks.

**Comment by AdamPawlinski** [comment]
I changed webkit-scrollbar to SfScrollable, but it is slightly different than on designs. The track is --c-light on design so should I change it in SfScrollable? Wdyt @pspaczek?

File: packages/shared/styles/components/organisms/SfSidebar.scss [hunk 1]
```diff
@@ -1,6 +1,7 @@
 @import "../../helpers";
 .sf-sidebar {
   &__aside {
+    --scrollable-max-height: var(--sidebar-content-max-height, 600px);
```
Comment (pspaczek) [inline packages/shared/styles/components/organisms/SfSidebar.scss:4]: @AdamPawlinski we can't use the static size. I will try to find some solution and give you feedback later

Comment (AdamPawlinski) [reply packages/shared/styles/components/organisms/SfSidebar.scss:4]: I removed this property and changed the way in which max scrollable height is calculated.

**Comment by mayashavin** [comment]
@AdamPawlinski can you please resolve the conflict? Thanks

File: packages/vue/docs/releases/v0.9.3.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Latest version - v0.9.2 🎉
```
Comment (mayashavin) [inline packages/vue/docs/releases/v0.9.3.md:1]: remove this file

File: packages/vue/src/components/organisms/SfSidebar/SfSidebar.stories.js [hunk 3]
```diff
@@ -59,7 +59,7 @@ storiesOf("Organisms|Sidebar", module)
         :class="customClass"
         :persistent="persistent"
       >
-        Total items: 0
+				Total items: 0
```
Comment (mayashavin) [inline packages/vue/src/components/organisms/SfSidebar/SfSidebar.stories.js:62]: update the story pls

**Review by mayashavin — CHANGES_REQUESTED** [review]
See my comments :)

File: packages/vue/src/components/organisms/SfSidebar/SfSidebar.stories.js (hunk 3 above)
Comment (AdamPawlinski) [reply packages/vue/src/components/organisms/SfSidebar/SfSidebar.stories.js]: updated thanks :)

File: packages/vue/docs/releases/v0.9.3.md (hunk 2 above)
Comment (AdamPawlinski) [reply packages/vue/docs/releases/v0.9.3.md]: Ok done.

**Review by mayashavin — APPROVED** [review]

**Pull request merged**